.goreleaser.yml

```diff
@@ -0,0 +1,37 @@
+---
+builds:
+  - binary: chproxy
+    goos:
+      - linux
+    goarch:
+      - amd64
```

Comment:
Why not building also for more Archs and OS ?

Reply:
I tested another ARCHs, and the best way to compile additional ARCHs is to have a dedicated build for each arch separately. So, I will add one by one in the next commits